packages/libllvm/lldb.subpackage.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_SUBPKG_INCLUDE="
+bin/lldb*
+include/lldb/
+lib/liblldb.so
+share/man/man1/lldb.1.gz
+"
+TERMUX_SUBPKG_DESCRIPTION="LLVM-based debugger"
```

Comment:
The only differences I could find from building this as a subpackage are that subpackages can't have their own homepage and they don't have a `LICENSE` file.